app/models/court_application_defendant_summary.rb

```diff
@@ -60,4 +62,8 @@ def defendant_organisation_name
   def offence_summary
     defendant.offences.map { |offence| OffenceSummary.new(offence_id: offence.id).to_builder.attributes! }
   end
+
+  def has_offences?
+    @court_application.court_application_type.has_offences
+  end
```

Comment:
so we are putting `have offences` in the `court_application_type`?

I'm surprised about that relationship, in the previous one `has_offences?` was checking the
`defendant_summary.offenceSummary`